src/main/kotlin/logic/use_cases/user/GetAllUsersUseCase.kt

```diff
@@ -6,7 +6,7 @@ import org.qudus.squad.model.entity.User
 class GetAllUsersUseCase(
     private val userRepository: UserRepository
 ) {
-    fun getAllUsers(): List<User> {
+    fun getAllUSers(): List<User> {
```

Comment:
Take care about the spelling
```suggestion
    fun getAllUsers(): List<User> {
```